static/main/main.js

```diff
@@ -7,12 +7,9 @@ process.env.PATH += ':/usr/local/bin';
 var mainWindow;
 
 app.on('ready', function () {
-
-    var atomScreen = require('screen');
-    var size = atomScreen.getPrimaryDisplay().workAreaSize;
     mainWindow = new BrowserWindow({
-        width: size.width,
-        height: size.height,
+        width: 700,
+        height: 450,
```

Comment:
I changed this because it makes the shell launch in full screen, not a good idea.
I think a better way is to set by default this values (700, 450) and then save the windows size that the users wants in the file .black-screen-state